src/app/components/layout/loading-content/loading-content.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="content">
+  <div *ngIf="isLoading">
+    <mat-spinner [color]="spinnerStyle" class="in-button big"></mat-spinner>
+    <p>Loading...</p>
+  </div>
+  <div *ngIf="!isLoading">
+    <i class="material-icons">announcement</i>
+    <p>There are still no transactions to show</p>
```

Comment:
The text should be obtained from ngx-translate.

Reply:
Fixed.